training/model_evaluation/src/image_classification_dataset.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""AzureML ACFT Image evaluation component - image classification dataset."""
+
+from __future__ import annotations
+import base64
+import json
+import pandas as pd
+
+from PIL import Image
+from typing import cast, Dict
+
+from azureml.automl.core.shared.constants import MLTableLiterals, MLTableDataLabel
+
+from azureml.acft.common_components import get_logger_app
+from azureml.acft.common_components.image.runtime_common.common import (
+    utils,
+)
+from azureml.acft.common_components.image.runtime_common.common.aml_dataset_base_wrapper import (
+    AmlDatasetBaseWrapper,
+)
+from azureml.acft.common_components.image.runtime_common.classification.io.read.dataset_wrappers import (
+    AmlDatasetWrapper,
+)
+
+from azureml.core import Workspace
+from azureml.core.run import Run
+
+logger = get_logger_app(__name__)
```

Comment:
Can you use our custom logger from logging utils instead of using another logger?

Reply:
Chaned to use logger from logging utils